image/base/mkosi.postinst

```diff
@@ -15,3 +15,19 @@ cp "${BUILDROOT}/usr/share/constellation/packagemanifest" "${OUTPUTDIR}/"
 
 # copy rpmdb to outputs
 cp "${BUILDROOT}"/var/lib/rpm/{rpmdb.sqlite-wal,rpmdb.sqlite-shm,rpmdb.sqlite,.rpm.lock} "${OUTPUTDIR}/"
+
+# FIXME(msanft):
+# Hack to satisfy Bazel's [output expectations](./BUILD.bazel).
```

Comment:
Did this work before the update?

Reply:
Yes. I assume it's due to a change in how `mkosi` calls `dnf` now, somehow not passing the variant to it. I spend quite some time looking at the [relevant code](https://github.com/systemd/mkosi/blob/v22/mkosi/installer/dnf.py) and couldn't pin down the exact reason though.